x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,431 @@
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "10d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST[continued]
+
+Now, version 2 of this policy has no `hot` phase, but if we run the Explain API again, we will see that nothing has changed.
+The index `my_index` is still executing version 1 of the policy.
+
+////
+[source,js]
+--------------------------------------------------
+GET my_index/_ilm/explain
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+////
+
+[source,js]
+--------------------------------------------------
+{
+  "indices": {
+    "my_index": {
+      "index": "my_index",
+      "managed": true,
+      "policy": "my_executing_policy",
+      "skip": false,
+      "lifecycle_date": 1538475653281,
+      "phase": "hot",
+      "phase_time": 1538475653317,
+      "action": "rollover",
+      "action_time": 1538475653317,
+      "step": "attempt_rollover",
+      "step_time": 1538475653317,
+      "phase_execution": {
+        "policy": "my_executing_policy",
+        "modified_date_in_millis": 1538475653317,
+        "version": 1,
+        "phase_definition": {
+          "minimum_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_docs": 1
+            }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/"lifecycle_date": 1538475653281/"lifecycle_date": $body.indices.my_index.lifecycle_date/]
+// TESTRESPONSE[s/"phase_time": 1538475653317/"phase_time": $body.indices.my_index.phase_time/]
+// TESTRESPONSE[s/"action_time": 1538475653317/"action_time": $body.indices.my_index.action_time/]
+// TESTRESPONSE[s/"step_time": 1538475653317/"step_time": $body.indices.my_index.step_time/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1538475653317/"modified_date_in_millis": $body.indices.my_index.phase_execution.modified_date_in_millis/]
+
+After indexing one document into `my_index` so that rollover succeeds and moves onto the next phase, we will notice something new.
+
+////
+[source,js]
+--------------------------------------------------
+PUT my_index/_doc/1
+{
+  "foo": "bar"
+}
+
+GET my_index/_ilm/explain
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+////
+
+[source,js]
+--------------------------------------------------
+{
+  "indices": {
+    "my_index": {
+      "index": "my_index",
+      "managed": true,
+      "policy": "my_executing_policy",
+      "skip": false,
+      "lifecycle_date": 1538475653281,
+      "phase": "warm",
+      "phase_time": 1538475653317,
+      "action": "forcemerge",
+      "action_time": 1538475653317,
+      "step": "forcemerge",
+      "step_time": 1538475653317,
+      "phase_execution": {
+        "policy": "my_executing_policy",
+        "modified_date_in_millis": 1538475653317,
+        "version": 2,
```

Comment:
I think we should add a callout here to highlight the fact that policy definition is showing version 2 now